setup.py

```diff
@@ -42,9 +42,11 @@
     include_package_data=True,
```

Comment:
Toctree captions are small (relative to the test), and in all caps. I think this is also how section headers used to be, too, but they were tweaked... can we also tweak toctree captions?

```rst
# Code to generate toctree with caption
.. toctree::
   :caption: Table of Contents
```

![Screenshot from 2021-07-16 12-27-57](https://user-images.githubusercontent.com/16181459/125979615-74d61c4f-7a83-4dc2-bfb6-8eaa2bf60644.png)

Reply:
Thanks, that's a great point! I changed it here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3355/commits/665784319432c3a29c692ce498b1534c57b515a4